### DISCUSSION TITLE
如何使用java代码调用 v2ray

### DISCUSSION THREAD

**Discussion by ABC15859888** [v2fly/v2ray-core#2393] [Q&A]
由于Java调用的接口需要代理  请教下 可以通过java 代码调用 v2ray 代理 http 请求嘛

**Comment by mydogshitgold** [comment]
Can you be more specific about what you're trying to solve?

Do you want to:
  - call the V2Ray API (golang) from Java?
  - start the V2Ray process from Java?
  - proxy Java http traffic through V2Ray?

**Reply by MyNanyh** [reply]
yes,Is it convenient to answer?

**Reply by moyuxitt** [reply]
how to proxy Java http traffic through V2Ray? where can i study this?
